### ISSUE TITLE
Voronoi tesselation on triangle surface mesh

### ISSUE THREAD

**Issue by yvanblanchard** [orzzzjq/gCentroidal-Voronoi-Tessellation#3]
Hello,
is it possible to apply your method from a triangle mesh (i.e. STL) ?
If yes, do you have a sample ?
Thank you.

**Comment by orzzzjq** [comment]
What do you mean by applying the method from a triangle mesh?

**Comment by yvanblanchard** [comment]
I mean, is it possible to generate the (Voronoi tessel) points starting from a STL triangle mesh (tesselation).
For example, this : 
![image](https://user-images.githubusercontent.com/63931618/166486476-74520b3b-cb5c-4d65-aabf-2988d34e4269.png)

**Comment by orzzzjq** [comment]
Yes, sure. That's a usual way to compute Voronoi tesselations, as Delaunay traingulation and Voronoi tesselation are dual of each other. I think there are many libraries that can achieve this, but this repo is not for Voronoi tesselation, it's *centroidal* Voronoi tesselation, and we compute Voronoi tesselations directly from the points.

**Comment by yvanblanchard** [comment]
Ok thank you.
Is it possible to compile the code with Visual Studio 2017, on Windows10 x64 ? The only dependency is CUDA,right ?

**Comment by orzzzjq** [comment]
Yes, you can compile it with VS, but this repo is not for pure Voronoi tesselation. If you want to compute Voronoi tesselation, can have a look at https://github.com/orzzzjq/Parallel-Banding-Algorithm-plus
